azure-devops/run_build.yml

```diff
@@ -23,22 +23,26 @@ jobs:
         filePath: $(Build.SourcesDirectory)/azure-devops/install_msvc_preview.ps1
     - task: CacheBeta@0
       displayName: Cache vcpkg
+      timeoutInMinutes: 5
       inputs:
         key: $(vcpkgResponseFile) | $(Build.SourcesDirectory)/.git/modules/vcpkg/HEAD
         path: '$(vcpkgLocation)'
     - task: run-vcpkg@0
       displayName: 'Run vcpkg to Install boost-build'
+      timeoutInMinutes: 5
       inputs:
         vcpkgArguments: 'boost-build:x86-windows'
         vcpkgDirectory: '$(vcpkgLocation)'
     - task: run-vcpkg@0
       displayName: 'Run vcpkg'
+      timeoutInMinutes: 5
       inputs:
         vcpkgArguments: '@$(vcpkgResponseFile)'
         vcpkgDirectory: '$(vcpkgLocation)'
         vcpkgTriplet: ${{ parameters.targetPlatform }}-windows
     - task: run-cmake@0
       displayName: 'Build Support Tools'
+      timeoutInMinutes: 5
       condition: eq('${{ parameters.targetPlatform }}', 'x64')
       inputs:
         cmakeListsTxtPath: 'tools/CMakeSettings.json'

```

Comment:
should this be `cmakeListsOrSettingsJson`?

Reply:
Billy said this was intentional: https://github.com/microsoft/STL/pull/163#discussion_r332309329